lms/services/course.py

```diff
@@ -73,17 +60,6 @@ def get(self, authority_provided_id):
             .one_or_none()
         )
 
-    def _create_legacy(self, authority_provided_id):
-        course = LegacyCourse(
-            consumer_key=self._application_instance.consumer_key,
-            authority_provided_id=authority_provided_id,
-            settings=self._new_course_settings(authority_provided_id),
-        )
-
-        self._db.add(course)
-
-        return course
-
```

Comment:
I think `_create` could be in-lined now. I'm not sure why it's taken out of the upsert method?

As a nice bonus if we alphabetically sort the methods all these internal methods which are for `upsert` will be next to it for easier reading.

I've also noticed `generate_authority_provided_id` is after a bunch of private methods which makes it easy to miss.